docs/tutorials/how_to_new_pokemon_1_10_0.md

```diff
@@ -0,0 +1,1153 @@
+
++   const u32 gMonFrontPic_Mewthree[] = INCBIN_U32("graphics/pokemon/mewthree/anim_front.4bpp.lz");
++   const u32 gMonBackPic_Mewthree[] = INCBIN_U32("graphics/pokemon/mewthree/back.4bpp.lz");
++   const u32 gMonPalette_Mewthree[] = INCBIN_U32("graphics/pokemon/mewthree/normal.gbapal.lz");
++   const u32 gMonShinyPalette_Mewthree[] = INCBIN_U32("graphics/pokemon/mewthree/shiny.gbapal.lz");
++   const u8 gMonIcon_Mewthree[] = INCBIN_U8("graphics/pokemon/mewthree/icon.4bpp");
++   const u8 gMonFootprint_Mewthree[] = INCBIN_U8("graphics/pokemon/mewthree/footprint.1bpp");
+```
+
+Please note that Pecharunt, the Pokémon that should be above your insertion for the time being, reads a `front.png` sprite instead of an `anim_front.png` sprite. This is because currently, Pecharunt lacks a 2nd frame. If the front sprite sheet of your species uses 2 frames, you should use `anim_front`.
+
+## 3. Add the animations to the rom
+
+You can define the animation order, in which the sprites will be shown. The first number is the sprite index (so 0 or 1) and the second number is the number of frames the sprite will be visible.
+
+Edit [src/data/pokemon_graphics/front_pic_anims.h](https://github.com/rh-hideout/pokeemerald-expansion/blob/master/src/data/pokemon_graphics/front_pic_anims.h):
+
+```diff
+#if P_FAMILY_PECHARUNT
+PLACEHOLDER_ANIM_SINGLE_FRAME(Pecharunt);
+#endif //P_FAMILY_PECHARUNT
+
++static const union AnimCmd sAnim_Mewthree_1[] =
++{
++    ANIMCMD_FRAME(1, 30),
++    ANIMCMD_FRAME(0, 20),
++    ANIMCMD_END,
++};
+```
+
+```diff
+#if P_FAMILY_PECHARUNT
+SINGLE_ANIMATION(Pecharunt);
+#endif //P_FAMILY_PECHARUNT
++SINGLE_ANIMATION(Mewthree);
+SINGLE_ANIMATION(Egg);
+```
+
+You might be wondering what `PLACEHOLDER_ANIM_SINGLE_FRAME` is. Well, since Pecharun only has 1 frame, we use what's called a preprocessor *macro* to have in a single line what otherwise would've been this in the C file:
+```c
+static const union AnimCmd sAnim_Pecharunt_1[] =
+{
+    ANIMCMD_FRAME(0, 1),
+    ANIMCMD_END,
+}
+```
+Instead, we can use the already established macro that does the same thing, replacing the value in parenthesis with what we want (in this case, `Pecharunt`):
+```c
+#define PLACEHOLDER_ANIM_SINGLE_FRAME(name)     \
+static const union AnimCmd sAnim_##name##_1[] = \
+{                                               \
+    ANIMCMD_FRAME(0, 1),                        \
+    ANIMCMD_END,                                \
+}
+```
+
+## 4. Linking graphic information to our Pokémon
+Now that we have all the external data ready, we just need to add it to `gSpeciesInfo` plus the rest of the animation and graphical data that we want to use:
+
+```diff
+ const struct SpeciesInfo gSpeciesInfo[] =
+ {
+     ...
+     [SPECIES_MEWTHREE] =
+     {
+        ...
+        .pokemonScale = 256,
+        .pokemonOffset = 0,
+        .trainerScale = 290,
+        .trainerOffset = 2,
++       .frontPic = gMonFrontPic_Mewthree,
++       .frontPicSize = MON_COORDS_SIZE(64, 64),
```

Comment:
This doesn't match the structure of the rest of these anymore, which are all in the form
```
        .frontPicSize = P_GBA_STYLE_SPECIES_GFX ? MON_COORDS_SIZE(56, 56) : MON_COORDS_SIZE(64, 48),
        .frontPicYOffset = P_GBA_STYLE_SPECIES_GFX ? 4 : 9,
        .backPicSize = P_GBA_STYLE_SPECIES_GFX ? MON_COORDS_SIZE(64, 48) : MON_COORDS_SIZE(56, 64),
        .backPicYOffset = P_GBA_STYLE_SPECIES_GFX ? 10 : 3,
```
what this means is probably worth clarifying

Reply:
It is clarified further down, and it wouldn't be relevant to a custom pokemon which doesn't have an original gen 3 sprite

Reply:
I am missing the clarification on `P_GBA_STYLE_SPECIES_GFX `, but if this isn't relevant to custom graphics anyways I'm okay with it being ignored :)